recipe/windows_cmake_install_bindir.patch

```diff
@@ -0,0 +1,37 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 3fd9d70..f8bf221 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -399,6 +399,7 @@ endif ()
+ if (OS_WINDOWS)
+   set (RUNTIME_INSTALL_DIR Bin)
+   set (LIBRARY_INSTALL_DIR Lib)
```

Comment:
No. Previously the `install` command was only using `LIBRARY_INSTALL_DIR` so everything was installed in `lib`. Since the `.dll` part of the library is considered a `RUNTIME` we needed to add different destinations for the `RUNTIME`, `LIBRARY` and `ARCHIVE` in the `install` command in order to have the `.dll` in `bin` and keep the `.lib` in `lib`. See https://cmake.org/cmake/help/v3.8/command/install.html#id3 for more details. Clearer now?